Umbraco-Cloud/Upgrades/Migrating-from-8-to-9/index.md

```diff
@@ -38,13 +38,13 @@ Should something fail during the migration, the Development environment can alwa
 
 * Import the database backup into SQL Server Management Studio
 
-* Clone down the Development environment from the Umbraco 9 Cloud project, test the project and make sure to login to the backoffice
+* Clone down the Development environment from the Umbraco 10 Cloud project, test the project and make sure to login to the backoffice
 
-* Update the connection string in the Umbraco 9 AppSetting.Json file so that it connects to the Umbraco 8 database
+* Update the connection string in the Umbraco 10 AppSetting.Json file so that it connects to the Umbraco 8 database
```

Comment:
Can we add an image here for reference?

Reply:
I've added a code snippet that should do the trick :D